docs/book/basic-usage.md

```diff
@@ -0,0 +1,43 @@
+# Basic Usage
+
+In the following example, a flash message is set in an edit action and the rendering is done after a redirect.
```

Comment:
Oh wow, I didn't realize we didn't have any docs at all :O 

Thanks for fixing this, @froschdesign - your patience is unrivaled!